standards/code-style.adoc

```diff
@@ -643,26 +650,30 @@ or `Fragment`.
 
 In the case of Activities the method is usually called `newStartIntent()`:
 
-[source, java]
+[source, kotlin]
 ----
-public static Intent newStartIntent(Context context, User user) {
-	Intent intent = new Intent(context, ThisActivity.class);
-	intent.putParcelableExtra(EXTRA_USER, user);
-	return intent;
+companion object {
+	fun newStartIntent(context: Context, user: User): Intent {
+	    val intent = Intent(context, ThisActivity::class.java)
+	    intent.putParcelableExtra(EXTRA_USER, user)
```

Comment:
One option for making this example use more idiomatic Kotlin would be to make use of an apply block.
```
val intent = Intent(context, ThisActivity::class.java).apply {
    putParcelableExtra(EXTRA_USER, user)
}
```

Reply:
Updated